rebar.config

```diff
@@ -2,11 +2,11 @@
 {plugins, [rebar3_hex]}.
 {deps, [
         lager,
-        {lager_logstash_backend, 
-         {git,"https://github.com/lambdaclass/lager_logstash_backend", {ref, "435f5a4"}}}
+        {lager_logstash_backend,
+         {git,"https://github.com/lambdaclass/lager_logstash_backend", {ref, "1ef21a9"}}}
```

Comment:
create a tag for the lager logstash backend called 0.1 and afterwards use the tag here. if you need to learn how to use git tags let me know